### PR TITLE
[Fix-16331] fix the resource file cannot be displayed hierarchically

### LINKED ISSUE #16331: [Bug] [dolphinscheduler-storage-hdfs] When you create a new task, the resource file cannot be displayed hierarchically
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

image
When creating a new task, the resource file cannot be displayed in a hierarchical manner, making it difficult to distinguish which file it is

### What you expected to happen

image
Ability to display files hierarchically

### How to reproduce

IN HdfsStorageOperator, 
```java
    public List<StorageEntity> listFileStorageEntityRecursively(String resourceAbsolutePath) {
        exceptionIfPathEmpty(resourceAbsolutePath);

        List<StorageEntity> result = new ArrayList<>();

        LinkedList<String> foldersToFetch = new LinkedList<>();
        foldersToFetch.addLast(resourceAbsolutePath);

        while (!foldersToFetch.isEmpty()) {
            String absolutePath = foldersToFetch.pollFirst();
            RemoteIterator<LocatedFileStatus> remoteIterator = fs.listFiles(new Path(absolutePath), true);
            while (remoteIterator.hasNext()) {
                LocatedFileStatus locatedFileStatus = remoteIterator.next();
                result.add(transformFileStatusToResourceMetadata(locatedFileStatus));
            }
        }
        return result;
    }
```

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by JohnZp**
## Purpose of the pull request

fix #16331 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/bin/start.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ DOLPHINSCHEDULER_HOME=$(cd ${BIN_DIR}/..;pwd)
 
 source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"
 
-JVM_ARGS_ENV_FILE=${BIN_DIR}/bin/jvm_args_env.sh
+JVM_ARGS_ENV_FILE=${DOLPHINSCHEDULER_HOME}/bin/jvm_args_env.sh
```
Comment (SbloodyS) [inline dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/bin/start.sh]: Please avoid unnessnary change.

**Review by SbloodyS — CHANGES_REQUESTED** [review]

**Review by Gallardot — COMMENTED** [review]
LGTM, but please make the CI happy.
```
mvn spotless:apply
```

**Comment by JohnZp** [comment]
> LGTM, but please make the CI happy.
> 
> ```
> mvn spotless:apply
> ```
Do I need to resubmit?

**Comment by SbloodyS** [comment]
> Do I need to resubmit?

Yes

**Review by ruanwenjun — CHANGES_REQUESTED** [review]
Since we may have the same folder under different tenant. 
e.g. 
/dolphinscheduler/tenant1/resources/spark/xx.jar
/dolphinscheduler/tenant2/resources/spark/yy.jar

If we show the folder here, there may exist two spark directory here? this will cause confusion.

**Comment by SbloodyS** [comment]
> e.g.
> /dolphinscheduler/tenant1/resources/spark/xx.jar
> /dolphinscheduler/tenant2/resources/spark/yy.jar

For this case, I think we can keep `tenant1/resources/spark/xx.jar` for user is enough.

Are you still working on this? @JohnZp

**Pull request closed without merging**